src/shapes/Polyline.ts

```diff
@@ -232,33 +237,21 @@ export class Polyline<
   /**
    * @override stroke and skewing are taken into account when projecting stroke on points,
    * therefore we don't want the default calculation to account for skewing as well.
-   * Though it is possible to pass `width` and `height` in `options`, doing so is very strange, use with discretion.
    *
    * @private
    */
-  _getTransformedDimensions(options: any = {}) {
+  _getTransformedDimensions(options: _getTransformedDimensionsParams = {}) {
     if (this.exactBoundingBox) {
       let size: Point;
       /* When `strokeUniform = true`, any changes to the properties require recalculating the `width` and `height` because
         the stroke projections are affected.
         When `strokeUniform = false`, we don't need to recalculate for scale transformations, as the effect of scale on
         projections follows a linear function (e.g. scaleX of 2 just multiply width by 2)*/
-      if (
-        Object.keys(options).some(
-          (key) =>
-            this.strokeUniform ||
-            (this.constructor as typeof Polyline).layoutProperties.includes(
-              key as keyof TProjectStrokeOnPointsOptions
-            )
-        )
-      ) {
+      if (this.strokeUniform || 'skewX' in options || 'skewY' in options) {
```

Comment:
there are 4 options and only skewX and skewY are  in layoutProperties, so we can check specifically.